src/platform/mouse.c

```diff
@@ -17,20 +17,19 @@ void system_mouse_get_relative_state(int *x, int *y)
 
 void system_mouse_set_relative_mode(int enabled)
 {
-    if (enabled) {
-        if (data.enabled) {
-            return;
-        }
+    if (enabled == data.enabled) {
+        return;
+    }
+    if(enabled) {
```

Comment:
One more `if(` :wink:

Reply:
I blame Visual Studio, which doesn't do as it's told!